apps/backend/src/modules/contributors/contributor.service.ts

```diff
@@ -102,6 +103,51 @@ class ContributorService implements Service {
 		return item.toObject();
 	}
 
+	public async merge(
+		currentContributorId: number,
+		{ selectedContributorId }: ContributorMergeRequestDto,
+	): Promise<ContributorGetAllItemResponseDto | null> {
+		if (currentContributorId === selectedContributorId) {
+			throw new ContributorError({
+				message: ExceptionMessage.CONTRIBUTOR_SELF_MERGE,
+				status: HTTPCode.CONFLICT,
+			});
+		}
+
+		const currentContributor =
+			await this.contributorRepository.find(currentContributorId);
+		const selectedContributor = await this.contributorRepository.find(
+			selectedContributorId,
+		);
+
+		if (!currentContributor || !selectedContributor) {
+			throw new ContributorError({
+				message: ExceptionMessage.CONTRIBUTOR_NOT_FOUND,
+				status: HTTPCode.NOT_FOUND,
+			});
+		}
+
+		try {
+			const mergedContributor = await this.contributorRepository.merge(
+				currentContributorId,
+				{
+					selectedContributorId,
+				},
+			);
+
+			if (!mergedContributor) {
+				return null;
+			}
+
+			return mergedContributor.toObject();
+		} catch {
+			throw new ContributorError({
+				message: ExceptionMessage.CONTRIBUTOR_MERGE_FAILED,
+				status: HTTPCode.CONFLICT,
```

Comment:
Are you sure the status is `CONFLICT`?

Reply:
I think conflict is ok here, it can be used sometimes for violation of database constraints